<?xml version="1.0"?>

<launch>
  <!-- Random move -->
  <node pkg="torob" type="random_goal" name="goal" output="screen">
      <param name="duration" value="5"/>
      <param name="min_distance" value="1.5"/>
      <param name="max_distance" value="3"/>
      <param name="max_angle" value="1.8"/>
  </node>
</launch>

